.github/ISSUE_TEMPLATE/meeting-agenda-template.md

```diff
@@ -0,0 +1,36 @@
+---
+name: 'Meeting Agenda: Create New'
+about: Use this template to create a new meeting agenda.
+title: Proposed SIG-Simulation meeting agenda for MMMM-DD-YY
+labels: mtg-agenda
+assignees: ''
+---
+
+## Meeting Details
+
+- **Date/Time:** Month Date, Year @ XX:XXpm UTC / XX:XXpm ET
```

Comment:
Aaarrgh, evil date format. Can we change this? It even uses a sensible one above.

Reply:
or YYYY/MM/DD, I'll change to that